### ISSUE TITLE
Need for a reverse function

### ISSUE THREAD

**Issue by lbenech** [shadowfaxtech/proSwipeButton#6]
Hello, great swipe button!

In addition, could it be possible to implement a "reverse()" method which allows to invert the button, I mean to be able to swipe from right to left?

Thanks in advance.

**Comment by ishaangarg** [comment]
Is it for RTL support or some other use case?

**Comment by im4eversmrt** [comment]
yes please , we really need that , i love this swipe button and dont really wanna ditch it because of RTL

**Comment by ishaangarg** [comment]
We'll be adding support soon.

In the meanwhile, you can checkout https://github.com/jefryjacky/proSwipeButton which is a fork of this repo with reverse functionality for RTL support.
